modules/Microsoft.DocumentDB/databaseAccounts/gremlinDatabases/deploy.bicep

```diff
@@ -50,7 +50,7 @@ resource databaseAccount 'Microsoft.DocumentDB/databaseAccounts@2022-02-15-previ
   name: databaseAccountName
 }
 
-var databaseOptions = contains(databaseAccount.properties.capabilities, 'EnableServerless') ? {} : {
+var databaseOptions = contains(databaseAccount.properties.capabilities, { name: 'EnableServerless' }) ? {} : {
```

Comment:
Hey @itpropro, did you test this? When I migrataed the original PR I had to change this as Bicep would throw syntax errors.

Reply:
Hey,
I tested the serverless deployment with Gremlin Mongo and SQL in my Azure subscription. That's where the fixes came from. I also checked the API specs and this is the actual representation and it was the only way I got it to work.